packages/lib/src/project/ProxyAdminProject.ts

```diff
@@ -47,11 +47,7 @@ class BaseProxyAdminProject extends BaseSimpleProject {
     contractParams: ContractInterface = {},
   ): Promise<Contract> {
     const { initMethod: initMethodName, initArgs } = contractParams;
-    const { implementationAddress, pAddress, initCallData } = await this._setUpgradeParams(
-      proxyAddress,
-      contract,
-      contractParams,
-    );
+    const { implementationAddress, pAddress } = await this._setUpgradeParams(proxyAddress, contract, contractParams);
```

Comment:
Are we renaming `proxyAddress` into `pAddress`? I'd keep the full name to be honest, it's easier to understand while reading the code.